### PR TITLE
Cu 865bzzx8r cards

### PR DESCRIPTION
**Pull request by janethtrs**
# Fixes ticket [#CU-865c5hc60](https://app.clickup.com/t/865c5hc60)

## Problem Description
Create card with the panel

## Checklist 
- [ ] Passes eslint linter 
- [ ] Passes backend integration tests 
- [ ] Added tests for this change 
- [ ] Added documentation for this change 
- [ ] Removed unused packages  

## Changes proposed in this pull request
The panel and card is done

## Reviewers
-  @sofia-gastelum 
-  @aviada-andreazayas 
- @aviada-rob  

## Additional Comments

### CODE REVIEW

File: src/app/components/Atoms/Panel/Panel.module.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.container {
```
Comment (aviada-andreazayas) [inline src/app/components/Atoms/Panel/Panel.module.css:1]: Delete empty style rule

File: src/app/components/Molecules/Card/Card.module.css [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.container {
+}
```
Comment (aviada-andreazayas) [inline src/app/components/Molecules/Card/Card.module.css:2]: Delete empty style rule

File: src/app/components/Atoms/Panel/Panel.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { Children } from 'react';
+import styles from './Panel.module.css';
+
+export interface IPanel {
+  width: number;
+  //height: number;
```
Comment (aviada-andreazayas) [inline src/app/components/Atoms/Panel/Panel.tsx:6]: Delete comment

File: src/app/components/Atoms/Panel/Panel.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  drop_shadow: string[];
+  children: any;
+  className: string;
+}
+
+const Panel: React.FC<IPanel> = ({ width,background_color,border_color,border_radius,border_weight,drop_shadow,children,className}) => {
+  return (
+    <div
+      style={{
+        //height: `${height}px`,
```
Comment (aviada-andreazayas) [inline src/app/components/Atoms/Panel/Panel.tsx:20]: Delete comment

**Review by aviada-andreazayas — CHANGES_REQUESTED** [review]

File: src/app/components/Molecules/Card/Card.module.css (hunk 2 above)
Comment (janethtrs) [reply src/app/components/Molecules/Card/Card.module.css]: Done

File: src/app/components/Atoms/Panel/Panel.tsx (hunk 3 above)
Comment (janethtrs) [reply src/app/components/Atoms/Panel/Panel.tsx]: Done

**Review by sofia-gastelum — APPROVED** [review]

**Review by aviada-andreazayas — APPROVED** [review]

**Pull request merged**